tests/Unit/DataStructures/Test_DataVector.cpp

```diff
@@ -10,42 +10,53 @@
 #include "tests/Unit/TestHelpers.hpp"
 
 SPECTRE_TEST_CASE("Unit.DataStructures.DataVector", "[DataStructures][Unit]") {
```

Comment:
Please don't just add scopes. It leads to horrible diffs and requires the reviewers to very carefully review every single line, and think about the change as a whole to make sure the efficacy of the test is unaltered.